devicon.json

```
@@ -3623,35 +3653,33 @@
           }
         ]
     },
-    {
-        "name": "spring",
+	{
+        "name": "subversion",
         "tags": [
-            "framework"
+            "svn",
+            "version"
         ],
         "versions": {
             "svg": [
                 "original",
-                "original-wordmark",
-                "plain",
-                "plain-wordmark"
+                "original-wordmark"
             ],
             "font": [
-                "plain",
-                "plain-wordmark"
+                "original",
+                "original-wordmark"
             ]
         },
-        "color": "#5FB832",
-        "aliases": []
-    },
-    {
-        "name": "spss",
-        "tags": ["package", "statistics"],
-        "versions": {
-          "svg": ["original", "plain"],
-          "font": ["plain"]
-        },
-        "color": "#cc1e4c",
-        "aliases": []
+        "color": "#809cc8",
+        "aliases": [
+            {
+                "base": "original",
+                "alias": "plain"
+            },
+            {
+                "base": "original-wordmark",
+                "alias": "plain-wordmark"
+            }
```

Since `wordmark` version doesn't fit being an icon, we don't need this.
```suggestion
            }
```